sdk/bazel_tools/runfiles/src/main/scala/com/daml/bazeltools/BazelRunfiles.scala

```diff
@@ -18,9 +18,11 @@ trait BazelRunfiles {
     )
 
   def rlocation(path: String): String =
-    if (inBazelEnvironment)
-      Runfiles.preload.unmapped().rlocation(MainWorkspace + "/" + path)
-    else
+    if (inBazelEnvironment) {
+      val runfiles: Runfiles.Preloaded = Runfiles.preload()
+      // TODO: Forward the source repository correctly
+      runfiles.withSourceRepository("").rlocation(MainWorkspace + "/" + path)
```

Comment:
the handling of runfiles changed under bzlmod, [see updated spec](https://github.com/bazelbuild/proposals/blob/53c5691c3f08011f0abf1d840d5824a3bbe039e2/designs/2022-07-21-locating-runfiles-with-bzlmod.md#2-extend-the-runfiles-libraries-to-take-repository-mappings-into-account). This change is sufficient for runfiles from daml requested within daml. Once module boundaries need to be crossed, a source repo location may need to be passed forward.